test/powershell/Language/Operators/RangeOperator.Tests.ps1

```diff
@@ -0,0 +1,135 @@
+Describe "Range Operator" -Tags CI {
+    Context "Range integer operations" {
+        It "Range operator generates arrays of integers" {
+            $Range = 5..8
+            $Range.count | Should Be 4
+            $Range[0] | Should BeOfType [int]
+            $Range[1] | Should BeOfType [int]
+            $Range[2] | Should BeOfType [int]
+            $Range[3] | Should BeOfType [int]
+        }
+
+        It "Range operator accepts negative integer values" {
+            {
+                -8..-5
+            } |Should Not Throw

```

Comment:
Please remove the should - next line do the same and Pester can catch a throw there correctly.